### PR TITLE
Added Java Solution to content/3_Silver/Intro_Bitwise.mdx

### PR DESCRIPTION
**Pull request by bSquared42**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/3_Silver/Intro_Bitwise.mdx [hunk 1]
```diff
@@ -407,6 +407,52 @@ int main() {
...
+			pref[i] += pref[i - 1];
+		}
+
+		int ans = 0;
+		st = new StringTokenizer(f.readLine());
+		for (int i = 0; i < n; i++) {
+			int num = Integer.parseInt(st.nextToken());
+			// Determine if the i-th term in Pascal's Triangle
+			// is odd by determining the number of 2s in the factorization of n
+			// choose k.
```
Comment (ryanchou-dev) [inline content/3_Silver/Intro_Bitwise.mdx:440]:
```
/*
 * multiline 
 * comments
 * like
 * this
 */
```

**Review by ryanchou-dev — COMMENTED** [review]
first contribution! 🎉 

can you also send ac screenshot?

File: content/3_Silver/Intro_Bitwise.mdx (hunk 1 above)
Comment (bSquared42) [reply content/3_Silver/Intro_Bitwise.mdx]: Here is the AC screenshot:

Screenshot 2024-08-04 at 4 26 34 PM

Thank you!

**Review by ryanchou-dev — APPROVED** [review]
nice! thank you for the quick response 👍

**Pull request merged**